devex/cross-nuget/README.md

```diff
@@ -0,0 +1,136 @@
+# Cross-Platform Open Enclave SDK NuGet Package
+
+This directory contains the scripts used to build a cross-platform NuGet package
+for the Open Enclave SDK. The resulting package may be used on Windows to build
+SGX enclaves as well as on Linux to build SGX and OP-TEE enclaves for the Scalys
+Grapeboard (LS-1012A) and QEMU ARMv8.
+
+The SGX builds are suitable both for SGX1 systems as well as for systems that
+support FLC. All SGX builds contain enclave libraries with and without LVI
+mitigations.
+
+## Environment Setup
+
+Two build systems are required: the first must run Windows Server 2019 with GUI
+and the second must run Ubuntu 18.04 LTS or later (Desktop and Server are both
+OK). These two environments may exist on physical systems or in virtual
+machines, it does not matter.
+
+This section explains how to set up both environments.
+
+### Windows Server 2019
+
+1. Install Windows Server 2019 with GUI.
+2. Download the SDK's [Windows prerequisites installation
+   script](https://github.com/openenclave/openenclave/blob/master/scripts/install-windows-prereqs.ps1).
+3. If present, comment out the line in the script that starts with:
+   ```powershell
+   New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters" [...]
+   ```
+4. Execute the script as follows:
+   ```powershell
+   Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process
+   .\install-windows-prereqs.ps1 -InstallPath C:/oe_prereqs -LaunchConfiguration SGX1FLC-NoIntelDrivers -DCAPClientType None
+   ```
+5. When prompted, reboot the system.
+
+### Ubuntu
+
+1. Run `setup-host.sh`, accepting the LXD initialization defaults.
+2. Log out and log back in.
+3. Run `setup-containers.sh` to create and start the containers used for
+   building; unlike Docker containers, these remain running in the background
+   and have persistent storage; think of them as virtual machines instead of
+   containers.
+4. Clone the SDK:
+   ```bash
+   git clone -b v0.11.0 --recursive --depth=1 https://github.com/openenclave/openenclave sdk
+   ```
+5. Open `scripts/ansible/oe-contributors-setup.yml` and comment out the step
+   that installs the Intel SGX driver because drivers cannot be installed inside
+   containers.
+6. Install the prerequisites in each container:
+   ```bash
+   # Enter the Xenial container.
+   lxc exec oepkgxenial su ubuntu
+
+   cd /host/path/to/sdk
```

Comment:
Path

Reply:
I tried this  $lxc config device add oepkgxenial myhomedir disk source=$HOME path=/dest/ 
But got an error not found. I am sure I am making a mistake, but have no knowledge in this area :)